### PR TITLE
qemu_storage: added map method

### PR DESCRIPTION
**Pull request by hereischen**
1. added qemu-img map wrapper
2. fixed some style issue
id: 1623745
Signed-off-by: Haotong Chen <hachen@redhat.com>

### CODE REVIEW

**Comment by hereischen** [comment]
https://github.com/autotest/tp-qemu/pull/1473

File: virttest/qemu_storage.py [hunk 1]
```diff
@@ -631,6 +633,24 @@ def amend(self, params, cache_mode=None, ignore_status=False):
         cmd_result.stderr = results_stderr_52lts(cmd_result)
         return cmd_result
 
+    def map(self, output="human", ignore_status=False):
```
Comment (balamuruhans) [inline virttest/qemu_storage.py]: can we use different method name ? something like `qemu_img_map()` ?

File: virttest/qemu_storage.py [hunk 2]
```diff
@@ -631,6 +633,24 @@ def amend(self, params, cache_mode=None, ignore_status=False):
...
+
+        :param output: string of output format(`human`, `json`)
+        :param ignore_status: whether to raise an exception when command
+                              returns =! 0 (False), or not (True)
+        """
+        cmd_list = [self.image_cmd, "map", ("--output=%s" % output)]
+        if os.path.exists(self.image_filename):
+            cmd_list.append(self.image_filename)
+            cmd_result = process.system_output(" ".join(cmd_list),
+                                               ignore_status=False).decode()
```
Comment (balamuruhans) [inline virttest/qemu_storage.py]: I think you have missed to pass ignore_status,
`ignore_status=ignore_status`

File: virttest/qemu_storage.py [hunk 3]
```diff
@@ -631,6 +633,24 @@ def amend(self, params, cache_mode=None, ignore_status=False):
...
+            return cmd_result
+        else:
+            raise exceptions.TestError("Image file %s not found",
+                                       self.image_filename)
```
Comment (balamuruhans) [inline virttest/qemu_storage.py]: `%s/,/%`

**Comment by balamuruhans** [comment]
@hereischen `map` is a keyword, so we can have different method name. otherwise code LGTM.

**Comment by hereischen** [comment]
Cases related to preallocation will not be developed, as they have been set to manual only.

**Pull request closed without merging**